Sources/AWSLambdaRuntimeCore/Documentation.docc/Getting-started.md

```diff
@@ -0,0 +1,260 @@
+
+    If this is your first time SwiftPM experience, I would encourage you to [learn more about it][SwiftPM].
+  
+### Step 3: Develop your Lambda
+
+Next open your `SquareNumber.swift` and create your function. As mentioned earlier, in this example we just want to square numbers although your function can do whatever you want.
+
+```swift
+import AWSLambdaRuntime
+
+struct Input: Codable {
+  let number: Double
+}
+
+struct Number: Codable {
+  let result: Double
+}
+
+@main
+struct SquareNumberHandler: LambdaHandler {
+  typealias Event = Input
+  typealias Output = Number
+
+  init(context: LambdaInitializationContext) async {
+
+  }
+
+  func handle(_ event: Input, context: LambdaContext) async throws -> Number {
+    Number(result: input.number * input.number)
+  }
+}
+```
+
+First we define an `Input` and `Output` struct, that both conform to `Codable`. This way we ensure that our Lambda accepts json input and creates json output. The `@main` annotation marks the Lambda Handler as our executable entrypoint. 
+
+### Step 4: Test your Lambda locally
+
+Before we deploy our Lambda to AWS, we want to ensure that it works locally. In Xcode open the *Edit Scheme Menu* for your target.
+
+![SquareNumber Edit Scheme Menu](edit-scheme-menu.png)
+
+And add the environment variable `LOCAL_LAMBDA_SERVER_ENABLED=true` to your *Run* settings.
+
+![SquareNumber Edit Scheme Menu](edit-scheme-menu-set-env.png)
+
+Now you can compile and run your Lambda with Xcode. You should see an output like this in your console:
+
+```bash
+2020-05-31T13:01:14+0200 info: LocalLambdaServer started and listening on 127.0.0.1:7000, receiving payloads on /invoke
+2020-05-31T13:01:14+0200 info: lambda lifecycle starting with Configuration
+  General(logLevel: info))
+  Lifecycle(id: 73023677192036, maxTimes: 0, stopSignal: TERM)
+  RuntimeEngine(ip: 127.0.0.1, port: 7000, keepAlive: true, requestTimeout: nil
+```
+
+You can now invoke your Lambda from your terminal with `curl`:
+
+```bash
+curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"number": 3}' \
+  http://localhost:7000/invoke
+```
+
+If you get the response,
+
+```json
+{"result": 9}
+```
+
+everything worked fine.
+
+### Step 5: Build your Code for the AWS Lambda Environment
+
+Your Lambda will be executed on the Amazon Linux 2 operating system. This is why we need to compile our code for this environment. To achieve this, we use Docker.
+
+Run the following command in the root folder of your project.
+
+```bash 
+$ docker run \
+    --rm \
+    --volume "$(pwd)/:/src" \
+    --workdir "/src/" \
+    swift:5.6-amazonlinux2 \
```

Comment:
This needs to be updated to work with `1.0.0-alpha.1`, also above in `Package.swift`.

> 'swift-aws-lambda-runtime': error: package 'swift-aws-lambda-runtime' is using Swift tools version 5.7.0 but the installed version is 5.6.3